packages/frontend/amp/components/topMeta/SeriesLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { headline } from '@guardian/pasteup/typography';
+import { css } from 'emotion';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+const seriesStyle = (pillar: Pillar) => css`
+    color: ${pillarPalette[pillar].main};
+    ${headline(2)};
+    font-weight: 900;
+    text-decoration: none;
+    margin-top: 10px;
+    display: block;
+`;
+
+// Returns a series link if possible, and attempt to return a section link as a fallback if provided
+export const SeriesLink: React.SFC<{
+    baseURL: string;
+    tags: TagType[];
+    fallbackToSection: boolean;
+    sectionLabel?: string; // required for fallback only
+    sectionUrl?: string; // required for fallback only
+    pillar: Pillar;
+}> = ({
+    baseURL,
+    tags,
+    fallbackToSection,
+    sectionLabel,
+    sectionUrl,
+    pillar,
+}) => {
+    const tag = tags.find(t => t.type === 'Blog' || t.type === 'Series');
+
+    if (!tag && !fallbackToSection) {
```

Comment:
Because there are four branches, this is probably a rare case where I'd advocate if/else or switch.

Reply:
I don't think this will work as a switch as we are not checking the case of the same expression.   I tried if/else but the linters won't allow it and prefer the if statements 🤷‍♀